docs/Usage/CellMarkers.md

```diff
@@ -0,0 +1,73 @@
+
+### Cell Markers
+
+A cell marker is a special inline comment that defines the lines that begin each cell. The inline comment must start with the comment character registered in the language's grammar file.
+
+- Python: `#`
+- R: `#`
+- Julia: `#`
+- Javascript: `//`
+- C++: `//`
+- Stata: `//`
+- Matlab: `%`
+
+#### Code Cells
+
+After the comment character, you need to add one space plus any of the following markers on the same line:
+
+- `%%`
+- `<codecell>`
+- `In[]` (you can have any number of digits or spaces in the brackets)
```

Comment:
Could maybe be worded better.

Reply:
also `%% codecell` is valid

Reply:
> Could maybe be worded better.

I did but I feel the original was better

> also `%% codecell` is valid, and technically people should not rely on `%%` pointing to code automatically.

I have appended codecell next to %%'s all occurrences, we shouldn't mention bad practices in the docs